### PR TITLE
Logout Implimantation

### PR DESCRIPTION
**Pull request by AbhijitP98**

### CODE REVIEW

File: src/sections/Header.js [hunk 1]
```diff
@@ -10,6 +10,11 @@ import Logo from './../assets/logo_cooper_thinner.png';
 import CLink from '../components/atoms/CLink';
 import ProfileImage from './../assets/zahir-namane-hwc7eIQiTCE-unsplash.png';
 import { makeStyles } from '@mui/styles';
+import { useState } from 'react';
```
Comment (NaveenReliable) [inline src/sections/Header.js:13]: Keep imports in proper order as per airbnb standards.

File: src/sections/Header.js [hunk 2]
```diff
@@ -88,9 +107,36 @@ const StyledInputBase = styled(InputBase)(({ theme }) => ({
 
 export default function Header({ isLogin = true }) {
     const classes = useStyles();
+    const dispatch = useDispatch();
+    const navigate = useNavigate();
+    const {authToken} = useSelector((state)=>state.auth)
     const accessToken = localStorage.getItem("accessToken");
     const sessionToken = sessionStorage.getItem("accessToken");
+    const [anchorEl,setAnchorEl] = useState(null);
+    const [logOutPopup,setLogOutPopup] = useState(false);
```
Comment (NaveenReliable) [inline src/sections/Header.js:116]: make this boolean name as confirmLogoutPopup

**Review by NaveenReliable — CHANGES_REQUESTED** [review]

File: src/sections/Header.js (hunk 1 above)
Comment (AbhijitP98) [reply src/sections/Header.js]: Changed to proper imports.

File: src/sections/Header.js (hunk 2 above)
Comment (AbhijitP98) [reply src/sections/Header.js]: Changed the name

**Review by NaveenReliable — APPROVED** [review]

**Pull request merged**